docs/config/build-options.md

```diff
@@ -130,6 +130,7 @@ This option allows users to override CSS minification specifically instead of de
 - **Default:** `false`
 
 Generate production source maps. If `true`, a separate sourcemap file will be created. If `'inline'`, the sourcemap will be appended to the resulting output file as a data URI. `'hidden'` works like `true` except that the corresponding sourcemap comments in the bundled files are suppressed.
+Recommended to keep false for local development, true for CI. 
```

Comment:
Why do you recommend to do this?

Reply:
You misunderstood. I don't know. I opened this up to find out if this is the official recommendation or not.

Reply:
I don't get what you mean. Is it written some where?

I guess the recommendation would be: If you don't need a sourcemap, set it to `false`. If you need a sourcemap, set it to `true`. 🤔